README.md

```diff
@@ -1,7 +1,81 @@
-# cjs_capstone
 
-## Adding service key 
+# CodeTranslate
+
+CodeTranslate is a one-stop site for developers to rapidly scale-up in unfamiliar languages without having to memorize syntax, follow tutorials, or read documentation. It is “Google Translate” for code: a developer can type code in a known language and see the translation for an unfamiliar language. 
+
+
+## Demo
+Check out our live demo at http://bit.do/codetranslate. The repository for the frontend of the demo can be found [here](https://github.com/cjoshea9/cjs_capstone_frontend). Please note that the demo is currently hosted for free on [Heroku](https://www.heroku.com/) and may take some time to load up.
+
+## API Documentation
+We welcome use of our public api so that you can integrate code translations into your own projects! Check out our [API Documentation](https://app.swaggerhub.com/apis-docs/jackdavidweber/CodeTranslate/1.0.0#/developers/translate) if you are interested.
+
+## Technical Setup
+
+- Clone this repository ([how to clone](https://docs.github.com/en/enterprise/2.13/user/articles/cloning-a-repository))
+- `cd cjs_capstone`
+- make sure you have all [prerequisites](#Prerequisites) installed
+- To run tests: `pytest`
+- To run api locally: `python app.py`
+
+### Prerequisites
+
+The main pre-requisite is that you have pip installed. Install [pip](https://pypi.org/project/pip/). Once you have pip installed, take the following steps:
+
+- make sure you are in the root directory
+- create a [virtual enviornment](https://docs.python.org/3/library/venv.html) in python 3
+- copy-paste the following into terminal:
+
+```
+$ pip install -r requirements.txt # for running server
+$ pip install -U pytest # for testing
+$ pip install yapf # for auto formatting
+```
+### Adding service key 
```

Comment:
Is this optional to run the server?

Reply:
This was added by @swalsh15 - any thoughts?

Reply:
This is optional this was if you wanted to play around with data collection

Reply:
I think it would be nice to add a sentence about this only being required for those situations.